### DISCUSSION TITLE
How to change style of Custom component from another custom component with variant.

### DISCUSSION THREAD

**Discussion by ankursehdev** [chakra-ui/chakra-ui#7105] [Q&A]
Hello, I am working on a big project. I will try to explain the situation here. 

**I have a multipart CARD component with 2 variants - Which work well.**  something similar to this:

```
const Card = defineMultiStyleConfig({
parts: [Card, CardMedia, CardContent],
  baseStyle: {
    display: 'flex',
    flexDirection: 'column',
  },
  variants: {
    regular: {
      //styles here
    },
    collections: {
     //styles here
    },
  },
  defaultProps: {
    variant: 'collections',
  },
})
```

Now, there is a different Section that has 2 variants - I need to use existing CARD in this but want different styles applied to it.

**In the Section, I need to change how Card, CardMedia, and CardContent Look. Can I do it through the theme?**

```
const BannerSection = defineStyleConfig({
  // The styles all Cards have in common
  baseStyle: {
    display: 'flex',
    flexDirection: 'column',
  },
  // Two variants: rounded and smooth
  variants: {
    inline: {
      /** CAN I REFER TO CARD, CARDMEDIA, CARDCONTENT HERE SOMEHOW? **/
      cardMedia: {
        //Some custom styles here for section only
      },
    },
    full: {
      /** CAN I REFER TO CARD, CARDMEDIA, CARDCONTENT HERE SOMEHOW? something like**/
      card: {
        border: `1px solid green`,
      },
    },
  },
  defaultProps: {
    variant: 'collections',
  },
});

```
**Please guide me if this is possible. Currently, I am unable to do this with this approach.** 

TIA

**Comment by veloware** [comment]
Ive also wondered the same, essentially you're trying to do equivalent of this in styled-components? (just with added variant conditions too)- 

```
const BannerSection = styled.div`
   ...
   ${Card}: {
      border: `1px solid green`,
   }
`
```

Just wondering if this helps? https://chakra-ui.com/docs/styled-system/component-style#consuming-multipart-style-config Though that may only work if BannerSection is part of the multipart component?

Otherwise im thinking you may just need to have another variant of Card?
